src/genpy/rostime.py

```diff
@@ -180,7 +180,9 @@ def __ne__(self, other):
     def __cmp__(self, other):
```

Comment:
Ah, I see what the issue is - the implementation of `__lt__` delegates to `__cmp__`